From dbf60fc1deb4b266e6cc37ddcffae36ea67a6472 Mon Sep 17 00:00:00 2001
From: Joris Offouga <offougajoris@gmail.com>
Date: Thu, 25 Apr 2019 17:35:27 +0000
Subject: [PATCH] linux-fslc/imx7d-pico: Disable panel

This feature causes the linux break if the board
is not powered by an external usb hub

Signed-off-by: Joris Offouga <offougajoris@gmail.com>
---
 arch/arm/boot/dts/imx7d-pico.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-pico.dtsi b/arch/arm/boot/dts/imx7d-pico.dtsi
index e84ec845f185..b0f74a861980 100644
--- a/arch/arm/boot/dts/imx7d-pico.dtsi
+++ b/arch/arm/boot/dts/imx7d-pico.dtsi
@@ -93,6 +93,7 @@
 		compatible = "vxt,vl050-8048nt-c01";
 		backlight = <&backlight>;
 		power-supply = <&reg_lcd_3v3>;
+		status = "disable";
 
 		port {
 			panel_in: endpoint {
-- 
2.17.1

